programs/clearing_house/src/controller/liquidation.rs

```diff
@@ -169,21 +163,22 @@ pub fn liquidate_perp(
         .checked_sub(worst_case_base_asset_amount_after)
         .ok_or_else(math_error!())?;
 
-    let (margin_ratio, oracle_price) = {
+    let (margin_ratio, oracle_price_data) = {
         let market = &mut market_map.get_ref(&market_index)?;
-        let oracle_price = oracle_map.get_price_data(&market.amm.oracle)?.price;
+        let oracle_price_data = oracle_map.get_price_data(&market.amm.oracle)?;
```

Comment:
kinda unrelated to this PR but should we change the struct name of oracle_price_data / OraclePriceData to just oracle or something simpler? so we can do oracle.price ?

Reply:
yea think in another pr when we do naming/cleanups thats a good idea to revisit
(just best pratice to wait til more prs are merged to do changes like this)